packages/model-viewer/src/features/scene-graph/material.ts

```diff
@@ -115,6 +116,36 @@ export class Material extends ThreeDOMElement implements MaterialInterface {
     return (this[$sourceObject] as GLTFMaterial).emissiveFactor!;
   }
 
+  setBaseColorMap(uri: string) {
+    if (this[$pbrMetallicRoughness] !== null) {
+      this[$pbrMetallicRoughness].setBaseColorMap(uri);
+    }
+  }
+
+  setMetallicRoughnessMap(uri: string) {
```

Comment:
I wonder if we should have these duplicated here or not. I can see both sides, but I tend toward keeping the API minimal. Do you have a strong opinion?

Reply:
removed